Seneca-CDOT/telescope
# Remove wiki-feed-parser, add logic to get feeds from supabase

## Issue This PR Addresses
Fixes #2861
Waiting for #3373 to be merged, then update tests
~~Waiting for #3361 & #3359~~



## Type of Change




## Description
- Rip out `wiki-feed-parser.js` => will need to update tests as well
- Add `SUPABASE_URL` & `SERVICE_ROLE_KEY` to `parser` in `docker-compose.yml`
- Add supabase client



## Steps to test the PR
- `pnpm install`
- `pnpm services:start parser`
- Check the `parser`'s container log, if feeds are being processed => it's good
- 
### How to run e2e test: 
- `pnpm services:start`
- Stop the parser container
- `pnpm jest:e2e parser`
- 


## Checklist
